src/spyglass/common/common_device.py

```diff
@@ -586,10 +607,16 @@ def _add_probe_type(cls, new_probe_type_dict):
                 "database, please change the corresponding Probe object in the "
                 "NWB file. Entering 'N' will raise an exception."
             )
-            val = input(
-                f"Do you want to add probe type '{probe_type}' to the database?"
-                + " (y/N)"
-            )
+            while True:
```

Comment:
This logic should probably go into #743 instead of here.

Reply:
No, that function seems fine. I will revert the commit here. I was just frustrated that hitting enter would cause things to fail.